### DISCUSSION TITLE
Is there a way to prevent CowBoy application against ClickJacking?

### DISCUSSION THREAD

**Discussion by rsenet** [ninenines/cowboy#1635] [Q&A]
Clickjacking is an attack that tricks a user into clicking a webpage element which is invisible or disguised as another element. This can cause users to unwittingly download malware, visit malicious web pages, provide credentials or sensitive information, transfer money, or purchase products online.

Is there a way to prevent CowBoy application against ClickJacking?

**Comment by essen** [comment]
Cowboy doesn't generate web pages so there's nothing in Cowboy related to that, HTML, Javascript or other. It isn't relevant to this project. The relevant component would be either the framework or the template engine that you are using on top of Cowboy.

**Reply by rsenet** [reply]
Thanks a lot essen !
